.gitignore

```diff
@@ -1,5 +1,7 @@
 __pycache__
 *.sql
 *.sqlite
+*.qdrep
 *.dict
-*.csv
\ No newline at end of file
+*.csv
+*.log
```

Comment:
Add an empty line. 

Also, we should not be blindly adding things to a git ignore. You should run tests in separate folder and add a .gitignore there with `*/` in it to ignore everything.